### PR TITLE
Record regular errors in datadog

### PR DESCRIPTION
**Pull request by benrudolph**
@snopoke thought this might be interesting as well

buddy: @czue

### CODE REVIEW

File: corehq/util/tests/test_datadog_utils.py [hunk 1]
```diff
@@ -17,3 +17,14 @@ class DatadogUtilsTest(SimpleTestCase):
 ], DatadogUtilsTest)
 def test_sanitize_url(self, url, sanitized):
     self.assertEqual(sanitize_url(url), sanitized)
+
+
+@generate_cases([
```
Comment (benrudolph) [inline corehq/util/tests/test_datadog_utils.py]: took a page out of your book :book:

File: corehq/util/log.py [hunk 2]
```diff
@@ -87,6 +90,13 @@ def get_context(self, record):
             'code': code,
         })
         if request:
+            sanitized_url = sanitize_url(request.build_absolute_uri())
+            log_counter(ERROR_COUNT, {
```
Comment (benrudolph) [inline corehq/util/log.py]: made sure to test this locally

File: corehq/util/tests/test_datadog_utils.py (hunk 1 above)
Comment (snopoke) [inline corehq/util/tests/test_datadog_utils.py]: :page_facing_up: :heavy_check_mark:

**Comment by snopoke** [comment]
cool. GTG :clock1: :construction:

**Comment by czue** [comment]
nice. one step on the slow march to disabling 500 emails.

**Pull request merged**